app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -0,0 +1,41 @@
+package co.netguru.baby.monitor.client.feature.debug
+
+import co.netguru.baby.monitor.client.application.scope.AppScope
+import io.reactivex.Observable
+import io.reactivex.functions.BiFunction
+import io.reactivex.subjects.PublishSubject
+import timber.log.Timber
+import java.text.DateFormat
+import java.util.*
+import javax.inject.Inject
+
+@AppScope
```

Comment:
didn't we switch to using `@Singleton` which is the same thing, but already included?

Reply:
Just following project convention here. But good catch